src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1005,22 +1006,22 @@ bool OutputStateMachineEngine::_GetOscSetClipboard(const std::wstring_view strin
                                                    std::wstring& content,
                                                    bool& queryClipboard) const noexcept
 {
-    const size_t pos = string.find(';');
-    if (pos != std::wstring_view::npos)
+    const auto parts = Utils::SplitString(string, L';');
+    if (parts.size() < 2)
     {
-        const std::wstring_view substr = string.substr(pos + 1);
-        if (substr == L"?")
-        {
-            queryClipboard = true;
-            return true;
-        }
-        else
-        {
-            return Base64::s_Decode(substr, content);
-        }
+        return false;
     }
 
-    return false;
+    const std::wstring_view substr = til::at(parts, 1);
+    if (substr == L"?")
+    {
+        queryClipboard = true;

```

Comment:
please drop the `else` case and un-indent the base64 return. It was hard to tell that you were deleting a redundant return.